app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  SQL_QUERY = "SELECT COUNT(*) FROM users
```

Comment:
You're missing a requirement: "you should write a query with a *subquery* for this task."